packages/client/wallets/aa/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@crossmint/client-sdk-aa",
-    "version": "0.0.15",
+    "version": "0.0.16",
```

this will automatically be updated when we make a release, so best to omit from PR and the release script will handle
```suggestion
    "version": "0.0.15",
```

---

mmmm intresting, I did not change this

---

Looks like it's picking up this commit: https://github.com/Crossmint/crossmint-sdk/pull/343/commits/5fcade2a59e449e3fcdcd176595b1ff08c7e1884 from the rebase

---

fixed when pointing target branch to main